src/components/layerpublisher/LayerPublisher.js

```diff
@@ -43,6 +45,36 @@ function info(fileName) {
   PubSub.publish(STATUS_INFO, `Loaded ${fileName}.`);
 }
 
+async function initRasterLayer(data) {
+  const channelNames = Object.keys(data);
+  const channelUrls = Object.values(data).map(d => d.tileSource);
+  const raster = {
+    channelNames,
+    domains: Object.values(data).map(d => d.range),
```

Comment:
Basically range is currently in the source, and I plan to update this in #486 . I want all the hardcoded logic consolidated so that we know what should go in the schema. This PR is a step in that direction, because there was a lot of nested/hidden hardcoded logic that I want to remove.